src/Common/Model/Address.php

```diff
@@ -235,12 +235,12 @@ public static function createFromArray(array $data)
 
         $adminLevels = [];
         foreach ($data['adminLevels'] as $adminLevel) {
-            if (empty($adminLevel['level'])) {
+            if (null === $adminLevel['level'] || 0 === $adminLevel['level']) {
                 continue;
             }
 
             $name = $adminLevel['name'] ?? $adminLevel['code'] ?? null;
-            if (empty($name)) {
+            if (null === $name || 0 === strlen($name)) {
```

Comment:
This is quicker. 

```suggestion
            if (null === $name || '' ===$name) {
```

In symfony's code base we compare to the empty string instead of verifying the length.

Reply:
Good to know! 👍 
See f00d9a1c6a93a7644816e3befea546d405af0bbe